src/main/kotlin/com/cognifide/gradle/aem/instance/tail/Log.kt

```diff
@@ -48,31 +56,42 @@ class Log(
         private const val LOG_PATTERN = "$TIMESTAMP\\s$LEVEL\\s$SOURCE\\s$MESSAGE"
 
         private const val DATE_TIME_FORMAT = "dd.MM.yyyy HH:mm:ss.SSS"
+        private const val PRINT_DATE_TIME_FORMAT = "yyyy-MM-dd HH:mm:ss.SSS"
```

Comment:
do not create separate constants for string and formatter. let's have only constant formatter.

Reply:
OK